test/kube_testing/k8s.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	podStartTimeout  = 1 * time.Minute
+	podStartTimeout  = 3 * time.Minute
```

Comment:
OK, we move these timeouts now and then.  I hope this does not make tests too slow.

Reply:
In current approach Azure pulls all required images from docker hub. But this affects only the first deploy as images are getting cached on AKS nodes by docker as usual.